pkg/kdk/version.go

```diff
@@ -0,0 +1,29 @@
+// Copyright © 2018 Cisco Systems, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kdk
+
+var (
+	Version       = "0.8.0"
```

Comment:
How does this comport with master branch's approach to versioning?

Reply:
This should be set to "undefined".  WIll resolve in a followup versioning PR.  Right now @rtluckie 's verioning code has been added on top of my prior PR. WIll PR to disambiguate.